src/video_core/shader_environment.cpp

```diff
@@ -57,6 +59,49 @@ static Shader::TextureType ConvertType(const Tegra::Texture::TICEntry& entry) {
     }
 }
 
+static std::string StageToPreffix(Shader::Stage stage) {
+    switch (stage) {
+    case Shader::Stage::VertexB:
+        return "VB";
+    case Shader::Stage::TessellationControl:
+        return "TC";
+    case Shader::Stage::TessellationEval:
+        return "TE";
+    case Shader::Stage::Geometry:
+        return "GS";
+    case Shader::Stage::Fragment:
+        return "FS";
+    case Shader::Stage::Compute:
+        return "CS";
+    case Shader::Stage::VertexA:
+        return "VA";
+    default:
+        return "UK";
+    }
+}
+
+static void DumpImpl(u64 hash, u64* code, u32 read_highest, u32 read_lowest, u32 initial_offset,
```

Comment:
```suggestion
static void DumpImpl(u64 hash, const u64* code, u32 read_highest, u32 read_lowest, u32 initial_offset,
```

Since the code isn't modified